app/commands.py

```diff
@@ -504,14 +504,21 @@ def populate_redis_template_usage(service_id, day):
     Recalculate and replace the stats in redis for a day.
     To be used if redis data is lost for some reason.
     """
+    # the day variable is set to midnight of that day
```

Comment:
Does this comment go with the REDIS_ENABLED line? probably move it a couple down?